recipes/psyclone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "psyclone" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/psyclone-{{ version }}.tar.gz
+  sha256: 7b369353942358afcb93b199ef2b11116d756cf9d671667ca95fa83fb31f0355
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - pyparsing
+    - fparser >=0.2.0
+    - configparser
+    - jsonschema
+    - sympy
+    - jinja2
+    - termcolor
+    - python-graphviz
+
+test:
+  imports:
+    - psyclone
+  commands:
+    - pip check
+    - psyclone --version  # [not win] CLI not available on Windows
+  requires:
+    - python {{ python_min }}
+    - pip
+    - setuptools-scm # Required for 'pip check' to pass fparser
```

Comment:
This should be fixed in the fparser feedstock. That is a missing (or poorly defined) dependency there.

Reply:
I've opened a pull request at https://github.com/conda-forge/fparser-feedstock/pull/20

Reply:
Workaround has been removed